shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -142,10 +142,15 @@ public void onFrameAvailable(@NonNull SurfaceTexture texture) {
           }
         };
 
+    @NonNull
+    public SurfaceTextureWrapper textureWrapper() {
```

Comment:
should this be defined in the interface `TextureRegistry.SurfaceTextureEntry` since it's used in the old `FlutterView` as well?

Reply:
`TextureRegistry.SurfaceTextureEntry` is a public API that is given to plugins.  Plugins need access to the raw `SurfaceTexture`.